cluster-autoscaler/clusterstate/clusterstate.go

```diff
@@ -976,30 +976,30 @@ func (csr *ClusterStateRegistry) GetClusterSize() (currentSize, targetSize int)
 	return currentSize, targetSize
 }
 
-func (csr *ClusterStateRegistry) handleOutOfResourcesErrors(currentTime time.Time) {
+func (csr *ClusterStateRegistry) handleInstanceCreationErrorsErrors(currentTime time.Time) {
```

Comment:
Just to be sure, did you mean to repeat the word "Errors" in this function name?

Reply:
Of course not. Thanks :)